RecipesLyricsMashup/RecipesLyricsMashupV2/RecipeLyricsMashup/RecipeLyricsMashup/Pages/Index.cshtml

```diff
@@ -3,9 +3,10 @@
 @{
     ViewData["Title"] = "Home page";
     var recipePuppyJson2 = ViewData["recipePuppyJson2"];
-    QuickTypeRecipe.Result[] recipePuppyJson = (QuickTypeRecipe.Result[]) ViewData["recipePuppyJson"];
+    QuickTypeRecipe.Result[] recipePuppyRecipies = (QuickTypeRecipe.Result[]) ViewData["recipePuppyRecipies"];

```

Comment:
Using "control" + "R" in visual studio you can refactor your project to change all references to an object when renaming. I renamed "recipePuppyJson" to "recipePuppyRecipies" becuase it contains a list of recipies. This is best practice in terms of naming.

Reply:
Good point on refactoring and naming.